mkdocs.yml

```diff
@@ -0,0 +1,77 @@
+site_name: jte documentation
+site_description: "Documentation for jte (Java Template Engine)"
+repo_url: https://github.com/casid/jte/
+site_url: https://jte.gg/
+theme:
+  icon:
+    repo: fontawesome/brands/github
+  language: en 
+  name: material
+  features:
+    - navigation.top
+    - navigation.footer
+    - navigation.instant
+    - search.highlight
+    - content.code.copy
+    - content.tabs.link
+    - content.code.select
+  palette:
```

Comment:
Could we set the primary and accent color here? I tested a little bit over here https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/ and teal looks very close to the jte brand color. I looks good and readable in both dark and light mode.

```
primary: teal
accent: teal
```

Reply:
Done in fa0f4f7.

Reply:
Thanks!